Modules/System/SharePoint/src/SharePointClient.Codeunit.al

```diff
@@ -268,6 +279,70 @@ codeunit 9100 "SharePoint Client"
+    /// <param name="TempBlob">The TempBlob that will be populated with the file content.</param>
+    /// <returns>True if the operation was successful; otherwise - false.</returns>
+    procedure DownloadFileContent(OdataId: Text; var TempBlob: Codeunit "Temp Blob"): Boolean
+    begin
+        exit(SharePointClientImpl.DownloadFileContent(OdataId, TempBlob));
+    end;
+
+    /// <summary>
+    /// Downloads a file to an InStream.
+    /// </summary>
+    /// <param name="ServerRelativeUrl">URL of the file to Download.</param>
+    /// <param name="FileInStream">The InStream that will be populated with the file content.</param>
+    /// <returns>True if the operation was successful; otherwise - false.</returns>
+    procedure DownloadFileContentByServerRelativeUrl(ServerRelativeUrl: Text; var FileInStream: InStream): Boolean
+    begin
+        exit(SharePointClientImpl.DownloadFileContentByServerRelativeUrl(ServerRelativeUrl, FileInStream));
+    end;
+
+    /// <summary>
+    /// Downloads a file to the client.
+    /// </summary>
+    /// <param name="ServerRelativeUrl">URL of the file to Download.</param>
+    /// <param name="FileName">Name to be given to the file on the client side. Does not need to match the server side name.</param>
+    /// <returns>True if the operation was successful; otherwise - false.</returns>
+    procedure DownloadFileContentByServerRelativeUrl(ServerRelativeUrl: Text; FileName: Text): Boolean
+    begin
+        exit(SharePointClientImpl.DownloadFileContentByServerRelativeUrl(ServerRelativeUrl, FileName));
+    end;
+
+    /// <summary>
+    /// Downloads a file to a TempBlob.
+    /// </summary>
+    /// <param name="ServerRelativeUrl">URL of the file to Download.</param>
+    /// <param name="TempBlob">The TempBlob that will be populated with the file content.</param>
+    /// <returns>True if the operation was successful; otherwise - false.</returns>
+    procedure DownloadFileContentByServerRelativeUrl(ServerRelativeUrl: Text; var TempBlob: Codeunit "Temp Blob"): Boolean
+    begin
+        exit(SharePointClientImpl.DownloadFileContentByServerRelativeUrl(ServerRelativeUrl, TempBlob));
+    end;
+
+    /// <summary>
+    /// Deletes a file.
+    /// </summary>
+    /// <param name="OdataId">The odata.id parameter of the file entity.</param>
+    /// <returns>True if the operation was successful; otherwise - false.</returns>
+    procedure DeleteFile(OdataId: Text): Boolean
+    begin
+        exit(SharePointClientImpl.DeleteFile(OdataId));
+    end;
+
+    /// <summary>
+    /// Deletes a file.
+    /// </summary>
+    /// <param name="ServerRelativeUrl">URL of the file to delete.</param>
+    /// <returns>True if the operation was successful; otherwise - false.</returns>
+    procedure DeleteFileByServerRelativeUrl(ServerRelativeUrl: Text): Boolean
+    begin
+        exit(SharePointClientImpl.DeleteFileByServerRelativeUrl(OdataId));
```

Comment:
Looks like a copy paste error.
Should be `ServerRelativeUrl`

Reply:
Thanks for your feedback. Fixed !! :)